File: js/sumGreaterThanFive.js
// Problem# 166
// Write a function that returns the sum of elements in an array greater than 5.
// https://edabit.com/challenge/zHutBxQaWvzhzGgxL

const sumFive = (arr) =>
  arr.filter((num) => num > 5).reduce((a, b) => a + b, 0);

module.exports = sumFive;

File: python/sum_greater_than_five.py
"""
Sum Greater Than Five

Write a function that returns the sum of elements greater than 5, in the given list.

https://edabit.com/challenge/eXRfoKp8m9Q6qvpRv
"""
from typing import List, Union

Number = Union[int, float]


def sum_five(lst: List[Number]) -> Number:
    return sum(num for num in lst if num > 5)
